snapshot.yaml

```diff
@@ -1,17 +1,17 @@
-resolver: lts-18.14
+resolver: lts-18.21
 
 packages:
   - AesonBson-0.4.0
   - direct-daemonize-3.1
   - evoke-0.2021.9.14
-  - hashable-1.3.3.0
+  - hashable-1.3.4.0
```

Comment:
There is an even newer version of this available, but I'm not sure if our dependencies can cope with the breaking change. https://hackage.haskell.org/package/hashable-1.4.0.2